vignettes/examples.Rmd

```diff
@@ -152,10 +209,320 @@ tfrmt(
   )
   
   ) %>% 
-  print_to_gt(demog_data)
+  print_to_gt(demog_data) %>% 
+  tab_options(
+    container.width = 900
+  )
+```
+
+
+# AE table
+
+For the adverse events (AE) table, we will use the `ae_data` analysis results data, which is also based on the CDISC pilot data. This dataset has two different row label columns, `AEBODSYS` and `AETERM`, for system organ class and preferred term, respectively. There are also two order columns which will be used to set the row order of the output. Because this table has column spanners, we have two column variables, `col2` and `col1` to define the hierarchy of columns; these can be used in lieu of a `col_plan`, if desired. Finally, there is a param column and a value column. For brevity, we will subset to AEs with >10% prevalence in the High Dose group.
```

Comment:
```suggestion
For the adverse events (AE) table, we will use the `ae_data` analysis results data, which is also based on the CDISC pilot data. This dataset has two different row label columns, `AEBODSYS` and `AETERM`, for system organ class and preferred term, respectively. There are also two order columns which will be used to set the row order of the output. Because this table has column spanners, we have two column variables, `col2` and `col1` to define the hierarchy of columns; these can be used in lieu of a `col_plan` with `span_structures()`, if desired. Finally, there is a param column and a value column. For brevity, we will subset to AEs with >10% prevalence in the High Dose group.
```

It is not in lieu of a col_plan, just without the span structures